packages/calico/extra/start-calico-felix.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -xe
+
+export NODENAME=`/opt/mesosphere/bin/detect_ip`
+export IP=`/opt/mesosphere/bin/detect_ip`
+mkdir -p /etc/calico
+mkdir -p /var/lib/calico
+
+cp /opt/mesosphere/etc/calico/calicoctl.cfg /etc/calico/calicoctl.cfg
+cp /opt/mesosphere/active/calico/etc/profile.yaml /var/lib/calico/profile.yaml
+/opt/mesosphere/bin/calicoctl apply -f  /var/lib/calico/profile.yaml
```

Comment:
Does it mean that if somebody removes our default policy and calico is restarted, the policy will be re-added?

Reply:
Yes. calicoctl handles input just like kubectl, and if you apply a definition, it will create/update this resource to now.

Reply:
This would mean that it is not possible to remove our default policy and that our customers may get confused at times. Imagine a situation when somebody removes the default policy and then after 2-3 months the server gets restarted - the policy appears again. 

Is product ok with that?

Reply:
Correct.
But we do not expose network policy functionality to our customers during this release, and  managing profile may be dangerous, this may be mentioned in calico doc, and calicoctl doc you are working on.

Reply:
I corrected this by applying the default profile when a node is initialized and Calico profile does not exist.

Reply:
Kubernetes achieves the default namespace-based network policy when a namespace is created, but we do not hold a concept like namespace.